apps/src/templates/maker/EligibilityConfirmDialog.story.jsx

```diff
@@ -10,8 +10,8 @@ export default storybook => {
         description: 'EligibilityConfirmDialog',
         story: () => (
           <EligibilityConfirmDialog
-            handleCancel={() => {}}
-            handleSubmit={() => {}}
+            onCancel={() => {}}
+            onSuccess={() => {}}
```

Comment:
Is there a rule for when to use 'handle' vs. 'on'?

Reply:
There isn't. I actually renamed this particular pair of props a few times while iterating on this, and semi by accident ended up with a pair that were almost the same as yours with on instead of handle.

I think what I generally go for is that props are name `onFoo` and the methods that you pass to them are `handleFoo`. However, this can quickly break down when you end up passing a method through multiple levels of the hierarchy.